internal/command/cmd.go

```diff
@@ -185,6 +186,7 @@ func (c *Cmd) buildCmd() (*exec.Cmd, context.CancelFunc) {
 	cmd.Dir = c.Dir
 	cmd.Stdout = c.Stdout
 	cmd.Stderr = c.Stderr
+	cmd.Env = c.Env
```

Comment:
In here mirror the same behavior as in the top level command.

i.e.

```go
cmd.Env = append(cmd.Environ, c.Env...)
```

The [Cmd.Env] should contain additional environment variables to be set not the `cmd.Environ` of the top level command